### ISSUE TITLE
Unsubscriptions count towards 15 alerts people get for free

### ISSUE THREAD

**Issue by mlissner** [freelawproject/courtlistener#2222]
People are able to sign up for a certain number of alerts for free. When we load the page, we check the server for how many alerts they have, and then we only allow them to make more if they don't have too many.

Well, we need to tweak this code, because it must be doing something like:

    select count(*) from alerts_table where user = foo;

It should be doing:

    select count(*) from alerts_table where user = foo and type = "subscription";

In other words, it's counting both subscriptions and unsubscriptions against the number that a person has. No good!
